input/charts.csv

```diff
@@ -1,27 +1,27 @@
-installedCapacity,3.05,Installed Capacity,timeSeries,,year,MW
-powerGeneration,3.06,Power Generation,timeSeries,,year,GWh
-cleanEnergyInvestment,investment,Clean Energy Investment,timeSeries,What's the volume of clean energy investment in the country over the last five years?,year,usd/m
\ No newline at end of file
+id,type,name,description,topic,indicatorId,labelX,labelY,unit
+utilityPrivatisation,answer,Utility privatisation,Sector of the power market open to private participation.,fundamentals,Utility privatisation,,,
+concentrationGeneration,answer,Concentration of generation market,Is the generation market concentrated?,fundamentals,Concentration of generation market,,,
+powerPlantFleet,answer,Power plant fleet modernisation program,Does the power generation fleet of the country need investment?,opportunities,Power plant fleet modernisation program,,,
+gapToTarget,answer,Gap to target,How large is the gap between the current level of renewables in the power sector and the country's closest target?,opportunities,Gap to target,,,
+upcomingAuctions,answer,Upcoming auctions,What is the volume of auctions announced in the country over the period 2018-2022 as a share of installed capacity?,opportunities,Upcoming auctions,,,
+wholesaleDistortions,answer,Wholesale price distortions,Are fossil fuels generators benefiting from subsidies that affect the price at which they can sell electricity onto the wholesale market?,fundamentals,Wholesale price distortions,,,
+sectorWholesale,answer,Power sector wholesale,,fundamentals,Wholesale power market,,,
+currencyVariation,answer,Currency variation,How high is the ten year variation of the local currency against the US dollar?,fundamentals,Currency variation,,,
+offtakerRisk,answer,Offtaker risk,How high is the offtake risk for independent generators?,fundamentals,Offtaker risk,,,
+standardisedPpa,answer,Standardised PPAs,Are power purchase agreements standardised in the market?,fundamentals,Standardised PPAs,,,
+ppaSufficientDuration,answer,PPAs of sufficient duration,Are PPAs of sufficient duration to be bankable for renewables developers?,fundamentals,PPAs of sufficient duration,,,
+purchaseObligation,answer,Purchase obligation,Is the offtaker obliged to pay for all the renewables output of generators?,fundamentals,Purchase obligation,,,
+minigridLicensing,answer,Mini-grids: licensing,Is it legally possible to set up a mini-grid or small power project incl. for self use?,fundamentals,Mini-grids: licensing,,,
+keroseneDieselSubsidies,answer,Kerosene and diesel subsidies,Are retail diesel and kerosene prices subsidised?,opportunities,Kerosene and diesel subsidies,,,
+curtailmentRisk,answer,Curtailment risk,Are renewables projects at risk of seeing their production curtailed?,fundamentals,Curtailment risk,,,
+foreignInvestment,absolute,Foreign investment,Disclosed share of foreign investment in the clean energy sector.,experience,Foreign investment,,,%
```

Comment:
Can we make the topic always plural: `experience` => `experiences`

Reply:
From @olafveerman:
> I'd rather not. 1) this is how it's referenced in other sheets such as subindicators, and 2) it would change it from a non-countable to a countable noun, and thus the meaning.